string color
string clase
float32 X
float32 Y
float32 R
int64 id
int64 numDetections